packages/evm-connector/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "@fuel-connectors/evm-connector",
+  "version": "0.1.1",
+  "type": "module",
+  "files": ["dist"],
+  "main": "./dist/wallet-connector-evm.umd.cjs",
+  "module": "./dist/wallet-connector-evm.js",
+  "exports": {
+    ".": {
+      "import": "./dist/wallet-connector-evm.js",
+      "require": "./dist/wallet-connector-evm.umd.cjs"
+    }
+  },
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "build:forc": "pnpm fuels-forc build --release --path ./signature-verification",
+    "build:resources": "pnpm run build:forc && tsup --dts-only ./src/utils/generatePredicateResources.ts",
+    "build": "pnpm run build:resources && tsup --dts-only",
+    "build:watch": "pnpm run build:resources && tsup --watch --dts-only",
+    "ts:check": "tsc --noEmit",
+    "test": "vitest"
+  },
+  "peerDependencies": {
+    "fuels": ">=0.77.0"
+  },
+  "dependencies": {
+    "@ethereumjs/util": "^9.0.1",
+    "@ethersproject/bytes": "^5.7.0",
+    "@fuel-ts/account": "^0.77.0",
+    "@fuel-ts/fuel-core": "^0.77.0",
+    "@fuel-ts/forc": "0.74.0",
```

Comment:
do we need these packages?

Reply:
`@fuel-ts/account` is required for `launchNodeAndGetWallets` in testing.
`@fuel-ts/forc` is required for the `pnpm fuels-forc` command.
Without  `@fuel-ts/fuel-core` the `beforeAll` in [the test file](https://github.com/FuelLabs/fuel-connectors/blob/56b18b355919c3f9e0fa4ad2172032d334530992/packages/evm-connector/src/test/walletConnector.test.ts#L51) times out

Reply:
We should move this deps to devDeps, if they are not needed on the connector